src/nxt_openssl.c

```diff
@@ -303,13 +303,13 @@ nxt_openssl_server_init(nxt_task_t *task, nxt_tls_conf_t *conf)
         goto fail;
     }
 */
-    ciphers = (conf->ciphers != NULL) ? conf->ciphers : "HIGH:!aNULL:!MD5";
-
-    if (SSL_CTX_set_cipher_list(ctx, ciphers) == 0) {
-        nxt_openssl_log_error(task, NXT_LOG_ALERT,
+    if (conf->ciphers) { /* else use system crypto policy */
+        if (SSL_CTX_set_cipher_list(ctx, conf->ciphers) == 0) {
+            nxt_openssl_log_error(task, NXT_LOG_ALERT,
                               "SSL_CTX_set_cipher_list(\"%s\") failed",
-                              ciphers);
-        goto fail;
+                              conf->ciphers);
+            goto fail;
+        }
     }
```

Comment:
Hi Remi!

Please detail in the commit message the mechanism by which system crypto policy is being used in the "else" case.  Is it by not calling `SSL_CTX_set_cipher_list()`, or how?

Reply:
If you don't call `SSL_CTX_set_cipher_list`, then you use system default. The goal of this PR